client/apps/settings/client/curations/venice_admin.coffee

```diff
@@ -0,0 +1,98 @@
+React = require 'react'
+ReactDOM = require 'react-dom'
+_ = require 'underscore'
+{ div, section, label, span, textarea, button } = React.DOM
+dropdownHeader = React.createFactory require '../../../edit/components/admin/components/dropdown_header.coffee'
+sectionFields = React.createFactory require './venice_section.coffee'
+
+module.exports = VeniceAdmin = React.createClass
+  displayName: 'VeniceAdmin'
+
+  getInitialState: ->
+    curation: @props.curation
+    activeSections: []
+    isChanged: false
+    isSaving: false
+
+  revealSection: (section) ->
+    sections = @state.activeSections
+    if section in @state.activeSections
+      sections = _.without(sections, section)
+      @setState activeSections: sections
+    else
+      sections.push(section)
+      @setState activeSections: sections
+
+  getActiveSection: (section) ->
+    active = if section in @state.activeSections then ' active' else ''
+    return active
+
+  isActiveSection: (section) ->
+    active = if section in @state.activeSections then true else false
+    return active
+
+  onChangeSection: (section, id) ->
+    @setState isChanged: true
+    newSections = @state.curation.get 'sections'
+    newSections[id] = section
+    @state.curation.set 'sections', newSections
+
+  onInputChange: (e) ->
+    @setState isChanged: true
+    @state.curation.set 'description', e.target.value
+
+  save: ->
+    @setState isSaving: true
+    @state.curation.save()
+    _.defer => @doneSaving()
```

Comment:
It looks like curation.save() has callbacks for done and error, but I couldn't manage to hook into these correctly, and ended up with defer instead.